### DISCUSSION TITLE
What would you recommend?  Hooks or New Site?

### DISCUSSION THREAD

**Discussion by TerryMiddleton** [datafeedr/wordpress-plugins#13] [Help]
Greetings from Austin, TX

We have a site with a theme that was built many many years ago.  It still works and we are scheduling out changing the theme, however it's going to be in the summer.

I've implemented datafeedr and it works.  Used he woocommerce_content() catch all and quickly understand (and appreciated) having a theme that is capable is best.  I read up on Hooks, which is an option, however while I could pull it off, not really interested in programming an old site.

So, I'm wondering if it's not just better to spin up a new site and house the datafeedr store under the new site.  

The new site will simply be for the shop.

I couple cons I can see would be I would have to manually put a menu item for SHOP on the content site.

I would not be able to easily integrate any of the shopping features (that I'm not even aware of yet) into my content site.

Am I missing anything?  I'm looking at the Premium offering.  Any reason why this wouldn't work or that  it's a bad idea?

Thanks for your insidght.

Terry

**Comment by EricBusch** [comment]
Hi Terry

According to WooCommerce docs, even your non-WC theme should work:

> WooCommerce looks great with all WordPress themes as of version 3.3, even if they are not WooCommerce-specific themes and do not formally declare support. Templates render inside the content, and this keeps everything looking natural on your site.

Personally I would not spin up a new site just for Datafeedr. You would lose the ability to, for example, write a blog post on your content site reviewing a product and then use the [WooCommerce shortcodes](https://docs.woocommerce.com/document/woocommerce-shortcodes/) in your post to display products from your store.

If it were me, I would use your hook method for now then when you change out your theme in the summer, your store's URLs and SERP rankings won't change. The only thing that will change is the design which is fine.

Hope this helps!

If you have any other questions, please don't hesitate to ask.

Thanks,
Eric
